.changeset/wet-peaches-grow.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/clerk-react': patch
+---
+
+Respect redirect options passed to buildSignInUrl, buildSignUpUrl, SignInButton and SignUpButton
```

Comment:
Let's provide some more insights of what this change actually means for the user and how behavior changes for them